plugins/recruiting/src/__model__.ts

```diff
@@ -192,4 +152,35 @@ function createVacanciesAppModel (S: Builder) {
   S.mixin(recruiting.class.Vacancy, presentation.mixin.DetailForm, {
     component: recruiting.component.Vacancy
   })
+
+  const states = {
+    rejected: { name: 'Rejected' },
+    applied: { name: 'Applied' },
+    hrInterview: { name: 'HR interview' },
+    testTask: { name: 'Test Task' },
+    techInterview: { name: 'Technical interview' },
+    offer: { name: 'Offer' },
+    contract: { name: 'Contract signing' }
+  }
+
+  fsm('Default developer vacancy', recruiting.application.Vacancies, [recruiting.mixin.WithCandidateProps])
+    .transition(states.applied, states.hrInterview)
```

Comment:
Should it for builder to be more clear to pass like

.transition(states.applied, [states.hrInterview, states.rejected ]) => So it will create two instances.

Reply:
Yeah, good point